apps/mark-scan/backend/src/custom-paper-handler/state_machine.ts

```diff
@@ -1425,19 +1425,32 @@ export function buildMachine(
   );
 }
 
+const HIGHLY_FREQUENT_EVENT_TYPES: Array<PaperHandlerStatusEvent['type']> = [
+  'COVER_STATUS',
```

Comment:
The other two make sense to me. Will excluding `COVER_STATUS` leave us without logs for cover open and close events, or are those covered elsewhere?

Reply:
Covered elsewhere! And in fact the `COVER_STATUS` log doesn't even include the open/close status. Per the PR description:

> `COVER_STATUS`: Just tells us that we polled cover status and does not include the cover status itself. The transitions from cover-closed to cover-open and vice versa are already covered elsewhere.

Look for the "Transitioned to" logs in Kofi's original PR: https://github.com/votingworks/vxsuite/pull/5360